frontend/src/index.tsx

```diff
@@ -3,12 +3,21 @@ import { createRoot } from "react-dom/client";
 import "./index.css";
 import * as serviceWorkerRegistration from "./serviceWorkerRegistration";
 import App from "App";
-
+import { Auth0Provider } from "@auth0/auth0-react";
+const AUTH0_CLIENT_ID = process.env.REACT_APP_AUTH0_CLIENT_ID || "clientid";
```

Comment:
Could you add an `.env.example` file with stub values?

Reply:
I can, but why?
If it will fail to load the real values, it can be assumed it will fail to load the stub ones, I think